docs/customization.rst

```diff
@@ -582,6 +582,10 @@ Configuration Options
 
         - The :python:`"icon"` field can be specifed as `any of the icons bundled with this theme`_
         - The :python:`"link"` field is simply the hyperlink target added to the icon specified.
+        
+        Optionally, custom text used for the icon's tooltip by specifying a :python:`"name"`
```

Comment:
Optionally, custom text may be specified for the icon's tooltip with the :python:`"name"` field.